topics/proteomics/tutorials/metaquantome-function/tutorial.md

```diff
@@ -0,0 +1,411 @@
+>    - {% icon param-file %} *"metaquantome expanded or filtered file"*: `metaQuantome:Filter` (output of **metaQuantome: filter** {% icon tool %})
+>    - *"Mode"*: `Functional analysis`
+>        - *"Ontology"*: `Gene Ontology (GO terms)`
+>    - *"Name of the control sample group of control (used as denominator for fold change evaluation)."*: `T2`
+>    - *"Statistical test to perform"*: `standard t-test (parametric)`
+>    - *"Perform paired tests"*: `Yes`
+>
+>
+> 2. **Rename** {% icon galaxy-pencil %} the output file to `metaQuantome:stat`.
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    > In this tutorial, we have used the sample T2 as the control group for fold change evaluation. The users have the ability to choose their own control sample group. The initial version of the metaQuantome stat module could only process 2 conditions at a time. However, the new update of metaQuantome has the ability to perform statistical analysis across multiple conditions.
+>    {: .comment}
+>
+{: .hands_on}
+
+# **Visualize your Data**
+
+The  outputs of the visualization module of metaQuantome are high-quality, publication-ready visualizations: barplots for the analysis of a single sample or experimental condition and differential abundance analysis, volcano plots, heatmaps, and principal components analysis for comparisons between two or more experimental conditions. 
+Here were are showing 2 visualizations: **Barplot and Volcano Plot**. The Heatmap and PCA plot for multiple conditions are under development.
+There are two outputs of the visualization tool : an **HTML file (figure) and a tabular output containing the plot data**.
+
+## *metaQuantome: visualize* : Bar Chart
+
+> ### {% icon hands_on %} Hands-on: Bar chart visualization of Functions in T2 sample.
+>
+> 1. {% tool [metaQuantome: visualize](toolshed.g2.bx.psu.edu/repos/galaxyp/metaquantome_viz/metaquantome_viz/2.0.0-0) %} with the following parameters:
+>    - {% icon param-file %} *"Tabular file from metaQuantome stats or metaQuantome filter"*: `metaQuantome:stat` (output of **metaQuantome: stat** {% icon tool %})
+>    - {% icon param-file %} *"Samples file"*: `metaQuantome:create samples_file` (output of **metaQuantome: stat** {% icon tool %})
+>    - *"Mode"*: `Functional analysis`
+>        - *"Plot type"*: `Bar Chart (bar)`
+>            - *"Mean intensity column name"*: `T2_mean`
+>            - *"GO category to restrict to"*: `Molecular Function`
+>            - *"Color for the bar fill"*: `Orange`
+>
+>    > ### {% icon comment %} Comment
+>    > The user has a choice to run Bar plot for T2, T4 and T7 to know the top 5 or 10 most differently expressed functions. Here we have restricted the GO category to Molecular Function but the user can choose Biological Processes or Cellular component as their category.
+>    > 
+>    {: .comment}
+>
+{: .hands_on}
+
+![T2_MF](../../images/T2-mf.png){: width="85%"}
```

Comment:
```suggestion
![T2_MF](../../images/T2-mf.png "Top five differentially expressed Molecular Function GO terms."){: width="85%"}
```

Reply:
I'm actually a bit confused here about what is being compared. In the last comment, it says the T2 sample is being used as the control group. If so, then which two samples are being compared in this plot showing differential expression? Sample T2 versus... ?